src/main/java/space/obminyashka/items_exchange/dto/ChildDto.java

```diff
@@ -26,6 +27,7 @@
     private long id;
     @NotNull(message = "{invalid.not-null}")
     private Gender sex;
+    @ValidChildAge(message = "{invalid.child.age}")
     @ApiModelProperty(required = true, example = "yyyy-MM-dd")
     @NotNull(message = "{invalid.not-null}")
     @PastOrPresent(message = "{invalid.past-or-present.date}")
```

Comment:
Is that annotation needed to be here or can be removed after your latest changes?

Reply:
This annotation allow us to validate fields.